src/screens/Home/Home.tsx

```diff
@@ -17,7 +19,7 @@ const Home = () => {
       lastName={"User"}
       content="Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor accusantium consequuntur fuga expedita reiciendis nemo, nesciunt assumenda cumque vel architecto itaque quibusdam deserunt, harum magni quam ab, quas alias ea."
       media="https://media.istockphoto.com/id/615398376/photo/new-york-city-nyc-usa.jpg?s=612x612&w=0&k=20&c=rlrsrt4jbORPDSOW5df06Ik_X_5iQo1rYQd53xSs4nw="
-      />
+      /> */}
```

Comment:
Instead of commenting these out you should remove them

Reply:
Oh, I meant to change that back